diff --git a/modules/features_ui/features_ui.admin.inc b/modules/features_ui/features_ui.admin.inc
index 56ac7dc..0dc71e0 100644
--- a/modules/features_ui/features_ui.admin.inc
+++ b/modules/features_ui/features_ui.admin.inc
@@ -119,7 +119,7 @@ function template_preprocess_features_assignment_configure_form(&$variables) {
 
   foreach ($form['title'] as $id => $element) {
     // Do not take form control structures.
-    if (is_array($element) && element_child($id)) {
+    if (is_array($element) && Element::child($id)) {
       $table[$id]['#attributes']['class'][] = 'draggable';
       $table[$id]['#weight'] = $element['#weight'];
 
