diff --git a/modules/features_ui/features_ui.admin.inc b/modules/features_ui/features_ui.admin.inc
index 0dc71e0..cc932bd 100644
--- a/modules/features_ui/features_ui.admin.inc
+++ b/modules/features_ui/features_ui.admin.inc
@@ -5,7 +5,6 @@
  * Administration functions for features.module.
  */
 
-use Drupal\Component\Utility\String;
 use Drupal\Core\Render\Element;
 use Drupal\Core\Template\Attribute;
 use Drupal\Core\Url;
diff --git a/modules/features_ui/src/Form/AssignmentConfigureForm.php b/modules/features_ui/src/Form/AssignmentConfigureForm.php
index 4d5cebc..4fe71d8 100644
--- a/modules/features_ui/src/Form/AssignmentConfigureForm.php
+++ b/modules/features_ui/src/Form/AssignmentConfigureForm.php
@@ -7,7 +7,7 @@
 
 namespace Drupal\features_ui\Form;
 
-use Drupal\Component\Utility\String;
+use Drupal\Component\Utility\SafeMarkup;
 use Drupal\Component\Utility\Unicode;
 use Drupal\Component\Utility\Xss;
 use Drupal\features\FeaturesManagerInterface;
@@ -208,7 +208,7 @@ class AssignmentConfigureForm extends FormBase {
       $enabled = isset($enabled_methods[$method_id]);
       $method = $assignment_info[$method_id];
 
-      $method_name = String::checkPlain($method['name']);
+      $method_name = SafeMarkup::checkPlain($method['name']);
 
       $form['weight'][$method_id] = array(
         '#type' => 'weight',
diff --git a/modules/features_ui/src/Form/FeaturesDiffForm.php b/modules/features_ui/src/Form/FeaturesDiffForm.php
index 99cc322..c6c0992 100644
--- a/modules/features_ui/src/Form/FeaturesDiffForm.php
+++ b/modules/features_ui/src/Form/FeaturesDiffForm.php
@@ -7,7 +7,7 @@
 
 namespace Drupal\features_ui\Form;
 
-use Drupal\Component\Utility\String;
+use Drupal\Component\Utility\SafeMarkup;
 use Drupal\Component\Utility\Xss;
 use Drupal\features\FeaturesAssignerInterface;
 use Drupal\features\FeaturesGeneratorInterface;
@@ -139,7 +139,7 @@ class FeaturesDiffForm extends FormBase {
                 'data' => array(
                   '#type' => 'html_tag',
                   '#tag' => 'h2',
-                  '#value' => String::checkPlain($package['name']),
+                  '#value' => SafeMarkup::checkPlain($package['name']),
                 ),
               ),
               '#attributes' => array(
@@ -233,7 +233,7 @@ class FeaturesDiffForm extends FormBase {
         'row' => array(
           'data' => array(
             '#type' => 'details',
-            '#title' => String::checkPlain($name),
+            '#title' => SafeMarkup::checkPlain($name),
             '#open' => TRUE,
             '#description' => array(
               'data' => $details,
diff --git a/modules/features_ui/src/Form/FeaturesEditForm.php b/modules/features_ui/src/Form/FeaturesEditForm.php
index 12047d4..61849ca 100644
--- a/modules/features_ui/src/Form/FeaturesEditForm.php
+++ b/modules/features_ui/src/Form/FeaturesEditForm.php
@@ -7,7 +7,7 @@
 
 namespace Drupal\features_ui\Form;
 
-use Drupal\Component\Utility\String;
+use Drupal\Component\Utility\SafeMarkup;
 use Drupal\Component\Utility\Xss;
 use Drupal\features\FeaturesAssignerInterface;
 use Drupal\features\FeaturesGeneratorInterface;
@@ -247,7 +247,7 @@ class FeaturesEditForm extends FormBase {
         '#name' => $method_id,
         '#value' => $this->t('!name', array('!name' => $method['name'])),
         '#attributes' => array(
-          'title' => String::checkPlain($method['description']),
+          'title' => SafeMarkup::checkPlain($method['description']),
         ),
       );
     }
@@ -355,7 +355,7 @@ class FeaturesEditForm extends FormBase {
         $count += count($component_info['options'][$section]);
       }
       $extra_class = ($count == 0) ? 'features-export-empty' : '';
-      $component_name = str_replace('_', '-', String::checkPlain($component));
+      $component_name = str_replace('_', '-', SafeMarkup::checkPlain($component));
 
       if ($count + $component_items_count > 0) {
         $element[$component] = array(
@@ -694,9 +694,9 @@ class FeaturesEditForm extends FormBase {
    *   the human label for the item
    */
   protected function configLabel($type, $key, $label) {
-    $value = String::checkPlain($label);
+    $value = SafeMarkup::checkPlain($label);
     if ($key != $label) {
-      $value .= '  <span class="config-name">(' . String::checkPlain($key) . ')</span>';
+      $value .= '  <span class="config-name">(' . SafeMarkup::checkPlain($key) . ')</span>';
     }
     if (isset($this->conflicts[$type][$key])) {
       // Show what package the conflict is stored in.
@@ -706,7 +706,7 @@ class FeaturesEditForm extends FormBase {
       if (isset($packages[$package_name])) {
         $package_name = $packages[$package_name]['machine_name'];
       }
-      $value .= '  <span class="config-name">[' . t('in') . ' ' . String::checkPlain($package_name) . ']</span>';
+      $value .= '  <span class="config-name">[' . t('in') . ' ' . SafeMarkup::checkPlain($package_name) . ']</span>';
     }
     return $value;
   }
diff --git a/modules/features_ui/src/Form/FeaturesExportForm.php b/modules/features_ui/src/Form/FeaturesExportForm.php
index dbbd3fe..be4287e 100644
--- a/modules/features_ui/src/Form/FeaturesExportForm.php
+++ b/modules/features_ui/src/Form/FeaturesExportForm.php
@@ -7,7 +7,6 @@
 
 namespace Drupal\features_ui\Form;
 
-use Drupal\Component\Utility\String;
 use Drupal\Component\Utility\Xss;
 use Drupal\Component\Utility\SafeMarkup;
 use Drupal\features\FeaturesAssignerInterface;
@@ -170,7 +169,7 @@ class FeaturesExportForm extends FormBase {
           '#name' => $method_id,
           '#value' => $this->t('!name', array('!name' => $method['name'])),
           '#attributes' => array(
-            'title' => String::checkPlain($method['description']),
+            'title' => SafeMarkup::checkPlain($method['description']),
           ),
         );
       }
@@ -250,7 +249,7 @@ class FeaturesExportForm extends FormBase {
     );
     // Use 'data' instead of plain string value so a blank version doesn't remove column from table.
     $element['version'] = array(
-      'data' => String::checkPlain($package['version']),
+      'data' => SafeMarkup::checkPlain($package['version']),
       'class' => array('column-nowrap'),
     );
     $overrides = $this->featuresManager->detectOverrides($package);
@@ -266,8 +265,8 @@ class FeaturesExportForm extends FormBase {
     foreach ($package['config'] as $item_name) {
       $item = $config_collection[$item_name];
       $package_config[$item['type']][] = array(
-        'name' => String::checkPlain($item_name),
-        'label' => String::checkPlain($item['label']),
+        'name' => SafeMarkup::checkPlain($item_name),
+        'label' => SafeMarkup::checkPlain($item['label']),
         'class' => in_array($item_name, $overrides) ? 'features-override' :
           (in_array($item_name, $new_config) ? 'features-detected' : ''),
       );
@@ -278,8 +277,8 @@ class FeaturesExportForm extends FormBase {
         $item = $config_collection[$item_name];
         $conflicts[] = $item_name;
         $package_config[$item['type']][] = array(
-          'name' => String::checkPlain($item_name),
-          'label' => String::checkPlain($item['label']),
+          'name' => SafeMarkup::checkPlain($item_name),
+          'label' => SafeMarkup::checkPlain($item['label']),
           'class' => 'features-conflict',
         );
       }
@@ -338,9 +337,9 @@ class FeaturesExportForm extends FormBase {
           'data' => array(
             '#type' => 'html_tag',
             '#tag' => 'span',
-            '#value' => String::checkPlain($label),
+            '#value' => SafeMarkup::checkPlain($label),
             '#attributes' => array(
-              'title' => String::checkPlain($type),
+              'title' => SafeMarkup::checkPlain($type),
               'class' => 'features-item-label',
             ),
           ),
@@ -349,8 +348,8 @@ class FeaturesExportForm extends FormBase {
           'data' => array(
             '#theme' => 'features_items',
             '#items' => $package_config[$type],
-            '#value' => String::checkPlain($label),
-            '#title' => String::checkPlain($type),
+            '#value' => SafeMarkup::checkPlain($label),
+            '#title' => SafeMarkup::checkPlain($type),
           ),
           'class' => 'item',
         );
